RestaurantReservation.API/Validators/Authentication/LoginRequestDtoValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using RestaurantReservation.Api.Contracts.Models.Authentication;
+using RestaurantReservation.API.Utilities;
+
+namespace RestaurantReservation.Api.Validators;
+
+public class LoginRequestDtoValidator : AbstractValidator<LoginRequestDto>
+{
+    public LoginRequestDtoValidator()
+    {
+        RuleFor(x => x.Username)
+            .NotEmpty().NotNull()
+            .WithMessage(ApiErrors.UserNameRequired);
```

Comment:
We initially set the error code in FluentValidation, which will be helpful for the frontend side to assert it 

.WithErrorCode(ApiErrors.UserNameRequired)